pulsar-broker/src/main/java/org/apache/pulsar/broker/admin/impl/NamespacesBase.java

```diff
@@ -831,8 +831,11 @@ protected void internalSetAutoTopicCreation(AsyncResponse asyncResponse,
         validatePoliciesReadOnlyAccess();
         if (autoTopicCreationOverride != null) {
             if (!AutoTopicCreationOverrideImpl.isValidOverride(autoTopicCreationOverride)) {
-                throw new RestException(Status.PRECONDITION_FAILED,
-                        "Invalid configuration for autoTopicCreationOverride");
+                // To enhance error tolerance, automatic error correction
+                if (!autoTopicCreationOverride.autoFix()) {
+                    throw new RestException(Status.PRECONDITION_FAILED,
+                            "Invalid configuration for autoTopicCreationOverride");
+                }
```

Comment:
I think we should provide an accurate error message, not introduce auto-fix here. It will make the API more clear and straightforward

Reply:
Fixed, PTAL :)